app/src/main/java/com/nervousfish/nervousfish/modules/database/GsonDatabaseAdapter.java

```diff
@@ -40,47 +34,27 @@
 @SuppressWarnings("PMD.TooManyMethods")
 public final class GsonDatabaseAdapter implements IDatabase {
 
-    private final static Type TYPE_CONTACT_LIST = new TypeToken<ArrayList<Contact>>(){}.getType();
-    private final static Type TYPE_PROFILE_LIST = new TypeToken<ArrayList<Profile>>(){}.getType();
+    private final static Type TYPE_CONTACT_LIST = new TypeToken<ArrayList<Contact>>() {
+    }.getType();
+    private final static Type TYPE_PROFILE_LIST = new TypeToken<ArrayList<Profile>>() {
+    }.getType();
```

Comment:
imo that shouldn't be a reason but fine 😕 Could you at least do it as follows to make it a little bit less ugly (if the execution line continues, it should start at a higher indentation)
```java
private final static Type TYPE_CONTACT_LIST =
            new TypeToken<ArrayList<Contact>>() {
            }.getType();
private final static Type TYPE_PROFILE_LIST =
            new TypeToken<ArrayList<Profile>>() {
            }.getType();
```